themes/inbeat/layouts/partials/site-footer.html

```diff
@@ -52,7 +52,7 @@
                                     Book a demo
                                 </a>
                             </li>
-                            <li><a href="mailto:david@inbeat.co" title="Send an e-mail">Send us a message</a></li>
+                            <!-- <li><a href="mailto:david@inbeat.co" title="Send an e-mail">Send us a message</a></li> -->
```

Comment:
@BashCloud Let's completely remove it and not just comment it (removing unnecessary codes)

Reply:
Done